_sass/layouts/_blog.scss

```diff
@@ -44,8 +45,8 @@
     width: 100%;
     margin-bottom: 0;
     display: inline-block;
-    background-size: $feature-image-size;
-    border-bottom: 1px solid $border-color;
+    background-size: cover;
```

Comment:
What's the trade off between the size and cover?

Reply:
None, unless someone want to change it.

`$feature-image-size: cover; // options include "cover", "contain", "auto"`

It was removed, I don't know when.